net/mosquitto/Portfile

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8; mode: tcl; indent-tabs-mode: nil -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+
+name                mosquitto
+version             1.4.14
+
+categories          net devel
+platforms           darwin
+license             {EPL-1.0 EDL-1.0}
+
+maintainers         nomaintainer
+
+description         Mosquitto is an open-source MQTT 3.1/3.1.1 broker
+
+long_description    \
+    MQTT is a machine-to-machine / "Internet of Things" connectivity protocol. \
+    Mosquito provides an open-source MQTT v3.1.1 broker (i.e., server) and \
+    both  C and C++ client libraries.
+
+
+homepage            https://mosquitto.org
+master_sites        http://mosquitto.org/files/source/
+
+checksums           rmd160 e3066723255dfffbf2f86fc4d37625c866072477 \
+                    sha256 156b1fa731d12baad4b8b22f7b6a8af50ba881fc711b81e9919ec103cf2942d1
+
+depends_lib         port:c-ares \
+                    port:tcp_wrappers \
+                    port:libwebsockets \
+                    port:openssl
+
+cmake.out_of_source yes
+configure.args      -DWITH_WEBSOCKETS=ON -DUSE_LIBWRAP=ON
+
+set mosquitto_user  mosquitto
+set mosquitto_group mosquitto
+
+add_users ${mosquitto_user} group=${mosquitto_group} shell=/bin/sh \
+    home=${prefix}/var/lib/${name} realname=Mosquitto\ MQTT\ Server
+
+startupitem.create  yes
+startupitem.init    \
+    "MOSQUITTO=${prefix}/sbin/mosquitto"
+startupitem.start    \
+    "su ${mosquitto_user} -c \"\${MOSQUITTO}\""
+startupitem.stop    \
+    "su ${mosquitto_user} -c \"\pkill -U ${mosquitto_user} mosquitto\""
+
+livecheck.type  regex
+livecheck.url   http://mosquitto.org/files/source/
```

Comment:
You can replace that with `${master_sites}`.

Reply:
Done.